draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java

```diff
@@ -616,6 +659,9 @@ private void initializeAttributes(AttributeSet attrs) {
     this.enableClickToMinimize =
         attributes.getBoolean(R.styleable.draggable_view_enable_click_to_minimize_view,
             DEFAULT_ENABLE_CLICK_TO_MINIMIZE);
+    this.enableTouchListener =
+        attributes.getBoolean(R.styleable.draggable_view_enable_touch_listener_view,
+            DEFAULT_ENABLE_TOUCH_LISTENER);
     this.attributes = attributes;
```

Comment:
You don't need to provide xml configuration for this variable, please remove this.